README.md

```diff
@@ -1,3 +1,9 @@
+> For linux you can just run
+> ```bash
+> curl -s https://raw.githubusercontent.com/masters-of-cats/a-new-hope/main/install.sh | bash -s
+> ```
+> if you like.
```

Comment:
"to download the virus, run....."

Reply:
well now you've ruined my evil plan i hope you are happy